src/main/java/longah/node/Transaction.java

```diff
@@ -87,4 +87,44 @@ public void updateBalances() throws LongAhException {
             member.subtractFromBalance(amount);
         }
     }
+
+    /**
+     * Checks whether the input member name is the owner of a transaction.
+     * @param memberName String representation of member name to check
+     * @return a boolean value determining whether the input name is the owner of the transaction
+     */
+    public boolean isOwner(String memberName) {
```

Comment:
Minor semantics but "Owner" of a transaction doesn't sounds quite right. Do you mind changing the term across the board to "Owed" instead.
Similar to previous comment, since we're referencing `String` type might be better to name the method as `isOwed` instead. Or even better would be to make the comparison using `Member` objects instead since you have implemented the `isEqual` method for `Member` class.
Let me work on my proposed changes first though, if you intend to work on the latter improvement.

Reply:
Done with regards the naming! Comparison between member objects isn't really that ideal taking into account that Java has no operator overloading. Would be oso an extra step for user i/o handling to map names into a member instance